prog/minio/minio_server_nexus.rb

```diff
@@ -44,6 +44,8 @@ def before_run
     when_destroy_set? do
       if strand.label != "destroy"
         hop_destroy
+      elsif strand.stack.count > 1
```

Comment:
clever, it's a bit like an exception, in that it's unwinding the stack. But it presumes there is no clean-up in the intermediate stack frames...a fact that is likely true for now, but is a hazard. We could later improve it by checking if the relevant prog has some kind of clean-up label (essentially, a form of "rescue" or "catch").

Reply:
....I suppose it could check for a "destroy" label for progs in the stack too, maybe we should at least interrogate our current ones and think about the quantity of hazardousness.

Reply:
I checked all "push" usages in our repo. Only MinioServer, PostgresServer and BootstrapRhizome uses push at the moment. First two uses it with self.class (thus has the same destroy label with the other progs in the stack) and the last one does not have destroy label.